scripts/checkout_recursive.py

```diff
@@ -1,216 +0,0 @@
-#!/usr/bin/env python
```

Comment:
CI used to depend on this, but I'm not sure what the current status is.

Reply:
As of right now, while rosdep is still present in "pr-cleanroom", this file is not.

(btw, cleanroom should be updated since a bunch of stuff there is Travis- or Xenial-specific and currently unused)